crypto/crypto/src/unit_tests/cryptohasher.rs

```diff
@@ -1,62 +1,84 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+//! Test file for the procedural macros CryptoHasher and LCSCryptoHash.
+
 use crate as libra_crypto;
 use crate::{
-    hash::{CryptoHash, CryptoHasher},
+    hash::{CryptoHash, CryptoHasher, LIBRA_HASH_SUFFIX},
     HashValue,
 };
-use libra_crypto_derive::CryptoHasher;
+use libra_crypto_derive::{CryptoHasher, LCSCryptoHash};
+use serde::{Deserialize, Serialize};
 use tiny_keccak::{Hasher, Sha3};
 
-/// This file tests the CryptoHasher derive macro defined in the crypto-derive
-/// crate. This macro, as the comments there indicate, is meant to derive an
-/// instance of CryptoHasher (see `crate::hash`) for any type, which
-/// domain-separation tag is precisely the fully-qualified name of the
-/// struct. Here, we create a struct, manually "compute" its tag and compare it
-/// to the generated one.
```

Comment:
It's unfortunate that we lose the comment here. People will not expect this
test to actually be for a different crate.

Reply:
I tried to summarize this comment in the new line above and with the realistic example `Foo` below.